apps/timetable/admin/index.html

```diff
@@ -1,3 +1,4 @@
+<!--[if IE 9]> <html class="lt-ie10"> <![endif]-->
 <html>
```

Comment:
I checked and all browsers except IE9 are ignoring the conditional and IE9 doesn't render the html twice. In order for the HTML to validate we'll need to write is like this afaik